charts/superstream/values.yaml

```diff
@@ -153,6 +153,29 @@ superstreamEngine:
     remoteSyslog: superstream-syslog
     port: 5514
     protocol: udp
+  kafkaAutoScaler:
```

Comment:
@bkochauri-memphis Please add additional comments to this section describing the new service and its abilities